### ISSUE TITLE
Eliminar librería vue form generator

### ISSUE THREAD

**Issue by rubendelblanco** [rubendelblanco/openrpg#37]
Está dando más probelmas que soluciones e impide tener pleno control sobre los campos de los formularios. Hay que eliminarlo de los recursos user y character (en edit ya está hecho) y sustituirlo por un formulario hecho "a mano" a partir de BootstrapVue

**Comment by danirod** [comment]
Le bajaría la prioridad a esto salvo que haya formularios concretos que se sepa que están dando problemas, porque nos da agilidad, como he podido comprobar personalmente al mover los componentes a OpenRPG-UI, ya que no hay que perder tiempo metiendo etiquetas.

**Comment by rubendelblanco** [comment]
Sí, no hay problema en dejarlo para el final. El problema con esta librería estriba en sus limitaciones a la hora de querer interactuar con el comportamiento de los campos del formulario, que no deja (o no he sido capaz). Por eso el formulario de los puntos de experiencia lo hice a mano pero es cierto que para formularios más simples, ayuda mucho.
